lib/index.js

```diff
@@ -148,6 +148,12 @@ module.exports = (function() {
       .sort(function(rel1, rel2){ //using order
         var index1 = order.indexOf( path.basename(rel1.attrib.Type) );
         var index2 = order.indexOf( path.basename(rel2.attrib.Type) );
+        // If the attrib.Type is not in the order list, go to the end of sort
+        // Maybe we can do it more gracefully with the boolean operator
+        if( index1 < 0 && index2 >= 0) return 1; // rel1 go after rel2
```

Comment:
Nit: can you format braces to be consistent with file. 
```suggestion
        if (index1 < 0 && index2 >= 0) return 1; // rel1 go after rel2
```